web-app/js/portal/details/ZAxisPickerContainer.js

```diff
@@ -42,12 +42,14 @@ Portal.details.ZAxisPickerContainer = Ext.extend(Ext.Container, {
             }
         });
 
+        this.zAxisPickerTitleContainer = new Ext.Container({
+            html: String.format("<h4>{0}</h4>", this.zAxisPickerTitle)
```

Comment:
coud you add a loading message instead? 
otherwise utilise `updateTitleContainer`just as a string builder called `formatZAxisPickerTitle`. The formatting is in two places